treeherder/perf/fixtures/performance_bug_templates.yaml

```diff
@@ -18,9 +18,10 @@
 
       On the page above you can see an alert for each affected platform as well as a link to a graph showing the history of scores for this test. There is also a link to a treeherder page showing the Talos jobs in a pushlog format.
 
-      To learn more about the regressing test(s), please see: https://wiki.mozilla.org/TestEngineering/Performance/Talos
-
-      For information on reproducing and debugging the regression, either on try or locally, see: https://wiki.mozilla.org/TestEngineering/Performance/Talos/Running
+      To learn more about:
+          1. sheriffs' context, please see: https://wiki.mozilla.org/TestEngineering/Performance/Sheriffing/Workflow#Context
+          2. talos test(s), please see: https://wiki.mozilla.org/TestEngineering/Performance/Talos
+          3. reproducing and debugging the regression, either on try or locally, see: https://wiki.mozilla.org/TestEngineering/Performance/Talos/Running
```

Comment:
Since so much of that template language is repeated, can you move it to another file and import it to the fixtures file?

Reply:
> Since so much of that template language is repeated, can you move it to another file and import it to the fixtures file?

It doesn't seem that treeherder used before to import fixtures into fixtures. How do you recommend doing this?

Reply:
I don't know that you can do that in a fixture. It just seems to be that there could be one template instead of multiple and the few things that are different (who to cc) are filled in dynamically like revisionHref, alertSummary, etc.

Maybe thinking about how to do this could be a follow-up bug. I don't want to block on these changes.